src/binwalk/modules/extractor.py

```diff
@@ -4,7 +4,11 @@
 
 import os
 import re
-import pwd
+
+# Windows Python does not have pwd module
+if os.name == 'posix':
+    import pwd
```

Comment:
I'd do
```suggestion
# Windows Python does not have pwd module
try:
    import pwd
except ImportError:
   pwd = None
```
to simplify further checks to `if pwd` instead of `os.name == 'posix'`.